qemu/tests/qemu_option_check.py

```diff
@@ -49,10 +50,10 @@ def get_device_option(qemu_binary, device_name):
         if device_support_option.exit_status:
             test.error("Oops, output status is wrong")
         if not re.findall(r"%s\.(.*)=(.*)" % device_name,
-                          device_support_option.stdout):
```

Comment:
Please try not to avoid `results_stdout_52lts`.
You can use `device_support_option.stdout_text`  or `device_support_option.stdout.decode()` (This one supports 52lts) for the text string output.

Or use even better use `process.system_output`:
````python
        device_support_option = process.system_output("%s -device %s,? 2>&1" %
                                            (qemu_binary, device_name),
                                            timeout=10,
                                            ignore_status=Flase, shell=True).deocde().strip()
        if not re.findall(r"%s\.(.*)=(.*)" % device_name, device_support_option):
            test.fail("Qemu option check Failed")
        logging.info("Qemu options check successful. output is:\n%s" % device_support_option)
````
It will report test error when `device_support_option` status code is not 0 as before.
The previous code is not good.